api/v1/mongodbcommunity_types.go

```diff
@@ -449,6 +449,12 @@ func (m MongoDBCommunity) MongoURI() string {
 	return fmt.Sprintf("mongodb://%s", strings.Join(members, ","))
 }
 
+// MongoSRVURI returns a mongo srv uri which can be used to connect to this deployment
+func (m MongoDBCommunity) MongoSRVURI() string {
```

Comment:
can we make sure to use the same style as in enterprise for docstrings?

A single sentence ending with a period, followed by additional strings if needed